packages/suite/src/components/suite/Labeling/components/Metadata/withEditable.tsx

```diff
@@ -101,47 +113,59 @@ export const withEditable = (WrappedComponent: React.FC) => ({
         return () => {
             window.removeEventListener('keydown', keyboardHandler, false);
         };
-    }, [submit, onBlur, props.originalValue, divRef, wrapperRef, touched]);
+    }, [submit, onBlur, props.originalValue, divRef, touched]);
 
     return (
         <>
             <WrappedComponent {...props}>
+                {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
```

Comment:
wouldnt be nicer to have this div as a styled-component? it may fix eslint issue as well

Reply:
Maybe... I did not have courage to try but it may work. Ok will try :scientist:

Reply:
Indeed it looks like it does not break anything and lets me remove this eslint ignore, but lets wait for tests..